portal-web/test/functional/com/liferay/portalweb/macros/GogoShell.macro

```diff
@@ -111,14 +111,14 @@ definition {
 			value1 = ${moduleStatus});
 	}
 
-	@summary = "Default summary"
+	@summary = "This will assert no ${outputContent}"
```

Comment:
The name is already misleading for the macro. But this should be "This will assert the output is not ${outputContent}"